<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="buttonEnterParagraph">
      <property name="text">
       <string>Ввод текст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonReadParagraph">
      <property name="text">
       <string>Чтение текста</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSaveArray">
      <property name="text">
       <string>Сохранение массива</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonReadArray">
      <property name="text">
       <string>Чтение массива</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSaveStruct">
      <property name="text">
       <string>Сохранение структуры в текстовый файл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonReadStruct">
      <property name="text">
       <string>Чтенеие структуры с текста</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSaveStructBinary">
      <property name="text">
       <string>Сохранение структуры в двоичный файл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonReadStructBinary">
      <property name="text">
       <string>Чтение структуры с двоичного файла</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSaveSentence">
      <property name="text">
       <string>Сохранение предложения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonReadSentences">
      <property name="text">
       <string>Чтение предложения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonExit">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
